plugins/aws_ec2/help.md

```diff
@@ -173,43 +286,69 @@ Removes one or more ingress rules from a security group. See [http://docs.aws.am
 
 ##### Input
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|dry_run|boolean|None|False|Checks whether you have the required permissions for the action, without actually making the request, and provides an error response.|None|
-|group_id|string|None|False|The ID of the security group.|None|
-|group_name|string|None|False|[Default VPC] The name of the security group.|None|
-|ip_permissions|[]ip_permission|None|True|The IP permissions for the security group rule.|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|dry_run|boolean|None|False|Checks whether you have the required permissions for the action, without actually making the request, and provides an error response|None|True|
+|group_id|string|None|False|The ID of the security group|None|sg-123456|
+|group_name|string|None|False|[Default VPC] The name of the security group|None|groupname|
+|ip_permissions|[]ip_permission|None|True|The IP permissions for the security group rule|None|["{ "from_port": 8080, "ip_protocol": "tcp", "ip_ranges" : [{ "cidr_ip": "10.20.0.0/27", "description": "Blah" }], "to_port": 8090}"]|
+
+Example input:
+
+```
+{
+  "dry_run": true,
+  "group_id": "sg-123456",
+  "group_name": "groupname",
+  "ip_permissions": [
+    "{ \"from_port\": 8080, \"ip_protocol\": \"tcp\", \"ip_ranges\" : [{ \"cidr_ip\": \"10.20.0.0/27\", \"description\": \"Blah\" }], \"to_port\": 8090}"
+  ]
+}
```

Comment:
Same comment for all `Example inputs` 😃 . Wouldn't it look much better this way?
```suggestion
{
  "dry_run": true,
  "group_id": "sg-123456",
  "group_name": "groupname",
  "ip_permissions": [
    {
      "from_port": 8080,
      "ip_protocol": "tcp",
      "ip_ranges": [
        {
          "cidr_ip": "10.20.0.0/27",
          "description": "Blah"
        }
      ],
      "to_port": 8090
    }
  ]
}
```